.gitmodules

```diff
@@ -25,9 +26,11 @@
 [submodule "src/external/spdlog"]
 	path = src/external/spdlog
 	url = https://github.com/gabime/spdlog.git
+	branch = v1.x
```

Comment:
remove this too

Reply:
for what, i indicate just what branch is used that all ^^